packages/core/package.json

```diff
@@ -23,6 +23,7 @@
   "scripts": {
     "build": "yarn build:cjs && tsc -p src/tsconfig.json --outDir ./dist/esm --module es2020 && tsc-alias -p src/tsconfig.json --outDir ./dist/esm && cp ../../build/esm-package.json ./dist/esm/package.json",
     "build:cjs": "tsc --build src && cp ../../build/cjs-package.json ./dist/cjs/package.json",
+    "circular-deps:check": "madge --circular dist/cjs --exclude \"/(?:Address)|(?:Serialization/(?:PlutusData)|(?:Certificates)|(?:Scripts)|(?:AuxiliaryData))/*\"",
```

Comment:
Why excluding these paths?
Or in other words: why allowing circular dependencies here?